tests.py

```diff
@@ -133,6 +133,12 @@ def test_unicode_censorship_5(self):
 
         self.assertEqual(profanity.censor(bad_text), censored_text)
 
+    def test_custom_words(self):
```

Comment:
Can you please also add another test case, to check if the default wordlist is not removed after calling `add_censor_words` ?